miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebChromeClient.kt

```diff
@@ -85,4 +90,66 @@ internal class MiniAppWebChromeClient(
     internal fun doInjection(webView: WebView) {
         webView.evaluateJavascript(bridgeJs) {}
     }
+
+    //region fullscreen video
+    @VisibleForTesting
+    internal var customView: View? = null
+    private var customViewCallback: CustomViewCallback? = null
+    private var originalOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
+    private var originalSystemUiVisibility = View.SYSTEM_UI_FLAG_VISIBLE
+    private val fullScreenFlag = View.SYSTEM_UI_FLAG_FULLSCREEN or
+            View.SYSTEM_UI_FLAG_HIDE_NAVIGATION or
+            View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY
+
+    override fun onShowCustomView(paramView: View?, paramCustomViewCallback: CustomViewCallback?) {
```

Comment:
Probably this could use some internal documentation for helping team developers to understand when this particular callback originates.